_config.yml

```diff
@@ -13,4 +15,4 @@ defaults:
     scope:
       path: ""
     values:
-      layout: "default"
+      layout: "default"
```

Comment:
Please update your editor to include eol @ the end of the file please. You can also change this setting in git-

https://help.github.com/en/github/using-git/configuring-git-to-handle-line-endings